operator/Containerfile

```diff
@@ -0,0 +1,8 @@
+FROM quay.io/operator-framework/ansible-operator:main
```

Comment:
need to pin this to the release instead of main

Reply:
I'm good with this!